interp/stdarg.ml

```diff
@@ -51,13 +51,4 @@ let wit_uconstr = make0 "uconstr"
 
 let wit_open_constr = make0 ~dyn:(val_tag (topwit wit_constr)) "open_constr"
 
-let wit_clause_dft_concl  =
-  make0 "clause_dft_concl"
-
-(** Aliases for compatibility *)
-
-let wit_integer = wit_int
-let wit_preident = wit_pre_ident
-let wit_reference = wit_ref
-let wit_global = wit_ref
-let wit_clause = wit_clause_dft_concl
+let wit_clause_dft_concl = make0 "clause_dft_concl"
```

Comment:
I am fine with the other changes, but I think that particular one is the wrong way. `clause_dft_concl` looks like a much uglier name than just `clause`.

Reply:
Amended, thanks!